src/client/css/partials/breaches.css

```diff
@@ -167,16 +167,48 @@
   transform: rotate(-180deg);
 }
 
-.breaches-table details > div {
+.breaches-table details > article {
   padding: var(--padding-md);
   border-top: 1px solid var(--gray-10);
-  color: var(--gray-40);
+  font-size: 0.875rem;
+}
+
+.breaches-table details > article a {
+  text-decoration: none; /* TODO: remove this when anchor tags are fixed with localization changes */
 }
 
-.breaches-table details > div a {
-  text-decoration: none; /* TODO: remove this when updated copy is added */
+.breaches-table details > article p:first-of-type {
+  margin-top: 0;
+}
+
+.breaches-table details .resolve-list {
+  list-style-position: outside;
+  margin-left: 1rem;
+  padding-left: 0;
+}
+
+.breaches-table details .resolve-list-item {
+  position: relative;
+  padding-right: 60px;
```

Comment:
Nit: It would be good if we could use variables where possible. Would `--padding-xl` work here as well or could we introduce additional variables for spacing if not?

Reply:
Yeah, agreed and good point.  This was special case, but I made it work.  Checkbox should be mostly consistent throughout the site, so I also added this to global.css